tests/Integration/HttpClient/HttpClientSmsUserTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Integration\HttpClient;
+
+use App\HttpClient\HttpClientSmsUser;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\HttpClient\Exception\ClientException;
+
+class HttpClientSmsUserTest extends KernelTestCase
+{
+    private ?HttpClientSmsUser $client;
```

Comment:
why is this nullable? you initialize it in the setup

Reply:
Right.